### ISSUE TITLE
Calendar integration

### ISSUE THREAD

**Issue by MaxSchlueter** [blockbasti/just_another_workout_timer#93]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
I would like to have a button besides a workout that allows me to schedule it in my calendar for a certain time.

**Additional context**
I like this feature a lot in the Transportr app which allows one to add an itinerary to a calendar.

Also, if you give me some pointers where to change the code I would be happy to implement this new feature.

**Comment by blockbasti** [comment]
Hey, thanks for the idea. 

I think an integration directly in the app via notifications would be better in the long run. But for now, a calendar integration would be alright. Once the creation of a schedule is in place, creating the notifications shouldn't be that hard.

A good start would be building a simple UI for selecting the schedule (weekdays+fixed time or different times per day) and saving that schedule in the workout file itself. I'm not decided whether this should be integrated in the workout builder or as a dialog accessible from the home screen. Either version is fine by me. After that is in place, we can focus on creating calendar entries (or notifications) and updating them when changing the schedule.

If you need some more pointers, feel free to contact me.

**Comment by MaxSchlueter** [comment]
That's true. For me, it would not only be the name of the event being prefilled, but also the date and time duration. I also use my calendar as a kind of log to see how consistent I have been. Having a button would reduce the friction quite a bit of adding such a workout event to my calendar.
